src/main/src/main/java/org/geoserver/catalog/CoverageReaderFileConverter.java

```diff
@@ -0,0 +1,80 @@
+/* (c) 2014 - 2020 Open Source Geospatial Foundation - all rights reserved
```

Comment:
New files just need  ``(c) 2020 Open Source Geospatial Foundation - all rights reserved``
There is no need to add the rest.

Reply:
Do you want the unnecessary openplans lines removed?

Reply:
@jodygarnett do we have to remove them?

Reply:
For new files, yes because they had nothing to do with OpenPlans (it would just make things more complicated to have them there).